lighthouse-core/test/chromium-web-tests/README.md

```diff
@@ -0,0 +1,61 @@
+# Chromium Web Tests
+
+This runs the Chromium webtests using the devtools integration tester.
+
+`webtests/` contains all of our Lighthouse webtests.
+
+## Run
+
+```sh
+yarn build-devtools
+yarn webtests
```

Comment:
FWIW I think `webtests` only makes sense for those well into Chromium land. Maybe something with `devtools` in it?

Reply:
you want `yarn devtools-webtest`?

if we succeed in having CI handle the rebaseline, and externals never have to run this, is the naming less important (and better to use chromium jargon for us chromies?)

Reply:
> you want `yarn devtools-webtest`?
> 
> if we succeed in having CI handle the rebaseline, and externals never have to run this, is the naming less important (and better to use chromium jargon for us chromies?)

That sounds better to me, but open to something shorter too.

I guess my issue is that in Lightouse-land, `webtests` is like a complete red herring to the purpose of the test :)

Reply:
true.. `test-devtools` and `update:test-devtools`  ?

Reply:
^sgtm!